README.md

```diff
@@ -43,13 +43,19 @@ cookiecutter gh:ssciwr/cookiecutter-cpp-project
 ```
 
 This will start an interactive prompt that will configure and generate your project.
+One of the prompts will ask you for a remote repository URL, so you should head to
+the Git hosting service of your choice (e.g. `github.com`) and add a new repository.
 
 # Configuration
 
 This cookiecutter accepts the following configuration options:
 
 * `project_name`: The human-readable name of the project, defaults to `My C++ Project`
-* `project_slug`: This will be the name of the generated directorym, defaults to `my-cpp-project`
+* `remote_url`: The remote URL for the newly created repository. This is not only used
```

Comment:
would it be possible to validate the `remote_url` at this point? e.g. check the remote repo exists?

Reply:
Yes and No. It is of course possible, but I have so far refrained from introducing external dependencies of the cookiecutter in order to reduce potential entry barriers. I will see whether something can be done with `urllib` solely.

Reply:
ok - I just tried a bogus url and it didn't crash, so that's good.
the `git push origin main` then fails of course, but at least that gives an error message that can be googled..

Reply:
@lkeegan There are several possiblities, but none works really good, so I am inclined to only document that it should be an empty repo:
* `urllib.request.urlopen` to check for existence: HTTPS URLs only, no private repos
* `git ls-remote` to check for emptiness: Hangs on wrong URLs, no private repos

Reply:
I guess the way to do this (if we decide to in the future) would be to first identify the hosting provider from the url, and then use the relevant API

Reply:
Yes, but that would most certainly require an external dependency, at least `requests` (I don't think `urllib` is meant to talk to REST APIs).